code/modules/atmospherics/machinery/components/unary_devices/outlet_injector.dm

```diff
@@ -17,6 +17,39 @@
 	///Rate of operation of the device
 	var/volume_rate = 50
 
+	///id of air sensor its connected to
+	var/chamber_id
+
+/obj/machinery/atmospherics/components/unary/outlet_injector/examine(mob/user)
+	. = ..()
+	. += "Link with Air Sensor using Multitool"
```

Comment:
This isn't how we write these.

This and the other one should be inside `span_notice`, and written as a complete sentence.

You should also add contextual screentips when you have a multitool in your hand: https://github.com/tgstation/tgstation/tree/master/code/datums/elements/screentips#datumelementcontextual_screentip_tools

Reply:
Done. Not sure about the English, so you can change it if you like